src/actions/clientActions.js

```diff
@@ -0,0 +1,22 @@
+import { CLIENT_CHANGE } from '../constants/actionTypes'
+
+function isSmallScreen () {
+  return global.window.innerWidth <= 768
+}
+
+function clientChange (smallScreen) {
+  return {
+    type: CLIENT_CHANGE,
```

Comment:
So far it creates a CLIENT_CHANGE action so it feels somehow accurate, but luckily it is not part of public API, so we can rename it later.